### DISCUSSION TITLE
Thought I'd share what I'm working on

### DISCUSSION THREAD

**Discussion by oaklandgit** [replit/kaboom#200] [Show and tell]
I'm porting a Pico 8 game I made last year over to Kaboom.

https://replit.com/@LarryStone/Tailpipe-9-12-2021?v=1

So far, I just have the player control working.

**Comment by slmjkdbtl** [comment]
Looking good! Looking forward to see what how it'll turn out. Do you have a link for the original PICO8 game?

**Reply by oaklandgit** [reply]
Thanks for checking it out @slmjkdbtl. Here's the Pico 8 version:
https://www.lexaloffle.com/bbs/?tid=35622

**Comment by oaklandgit** [comment]
https://user-images.githubusercontent.com/421615/133912992-44003a9c-80bf-496f-98cf-659450ca89f2.mov

Decided to go with a bit of a 1-bit "Dark Castle" approach to the graphics.

**Reply by slmjkdbtl** [reply]
Love the art style so much ❤️

**Reply by oaklandgit** [reply]
Thank you!
